python/podio/sio_io.py

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env python3
 """Python module for reading sio files containing podio Frames"""
 
-from podio.base_reader import BaseReaderMixin
 
 from ROOT import gSystem
-gSystem.Load('libpodioSioIO')  # noqa: 402
+ret = gSystem.Load('libpodioSioIO')  # noqa: 402
+# Return values: -1 when it doesn't exist and -2 when there is a version mismatch
+if ret < 0:
+  raise ImportError('Error when importing libpodioSioIO')
 from ROOT import podio  # noqa: 402 # pylint: disable=wrong-import-position
-
-Writer = podio.SIOFrameWriter
+from podio.base_reader import BaseReaderMixin  # pylint: disable=wrong-import-position
```

Comment:
Is there a specific reason why this import has been moved here?

Also I don't think the `Writer` should be removed (definitely not as part of this PR).

Reply:
I mean I don't have anything against moving the `import` per se. It is just that the combination of moving it and then marking it with `disable=wrong-import-position` seems like a "double kill" to me. 

Maybe we should provide a virtualenv or something that has the same versions of `flake8` and `pylint` (and maybe even `pre-commit`) to make sure things can be run consistently in CI and locally.